diff -Naur a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -72,6 +72,8 @@ pcsx_rearmed_REVISION = 131a1b41835bc0eba3e35998dce376163a0a1b14
 
 picodrive_REPO = https://github.com/irixxxx/picodrive
 picodrive_MAKEFILE = Makefile.libretro
+# the next commit breaks picoarch-miyoomini build
+picodrive_REVISION = 97232a47c9bcf6005ee34af0eaff017c5d363eee
 
 pokemini_REPO = https://github.com/libretro/PokeMini
 pokemini_MAKEFILE = Makefile.libretro
